internal/commands/user/inputs.go

```diff
@@ -0,0 +1,73 @@
+package user
+
+import (
+	"errors"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	userDataEmail = "email"
+	userDataName  = "name"
+)
+
+// usersInputs are the filtering inputs for a user command
+type usersInputs struct {
+	State         realm.UserState
+	ProviderTypes []string
+	Pending       bool
+	Users         []string
+}
+
+// ResolveUsers will use the provided Realm client to resolve the users specified by the realm.App through inputs
+func (i *usersInputs) ResolveUsers(ui terminal.UI, client realm.Client, app realm.App) ([]realm.User, error) {
+	filter := realm.UserFilter{
+		IDs:       i.Users,
+		State:     i.State,
+		Pending:   i.Pending,
+		Providers: realm.StringSliceToProviderTypes(i.ProviderTypes...),
+	}
+	foundUsers, usersErr := client.FindUsers(app.GroupID, app.ID, filter)
+	if usersErr != nil {
+		return nil, usersErr
+	}
+	if len(i.Users) > 0 {
+		if len(foundUsers) == 0 {
+			return nil, errors.New("No users found")
```

Comment:
don't capitalize the error strings, we're often appending the messages and keep most things in lower case